docs/source/recipes/customizing-components.md

```diff
@@ -53,61 +53,64 @@ amendments required.
 Locate the `Tags.jsx` file and override this file so that there is a label in front of the tags with: `Tags:`.
 
 ```jsx hl_lines="20"
```

Comment:
It appears that this code highlighting directive is not being honored. Let's fix it!

```suggestion
```{code-block) jsx
:emphasize-lines: 20
```

This line appears before the change in the code block, so it should still highlight this line:

```
      Tags:
```

https://github.com/plone/volto/pull/3266/files#diff-b8133ecc81246c0e39589e6ca0088d0b3b443bfb3cda9f055946e6b9965e7933R75